File: Team_stats.js
//JS server to print out NBA team information

const config = require('./config.json');
const commands = require('./NBA_Stats');

console.log(commands);

const options = {
	method: 'GET',
	headers: {
		'X-RapidAPI-Key': config.headers['X-RapidAPI-Key'],
		'X-RapidAPI-Host': config.headers['X-RapidAPI-Host']
	}
};

module.exports = async function getTeamInfo() {
    console.log("TEST");
    fetch(config.url + '/teams?id=2', options)
	.then(response => response.json())
	.then(parsed_data => {
        console.log("HELLO");
        console.log(parsed_data['response'][0]);
        //console.log(parsed_data['response'][0]['name']);
        message.channel.send(`Team name: ${parsed_data['response'][0]['name']}`);
        message.channel.send(`Team Logo: ${parsed_data['response'][0]['logo']}`);
    })
	.catch(err => console.error(err));
}

File: Team_stats.py
import requests
import discord
import json
import sys

# Function to format received json file
def jprint(obj) :
    text = json.dumps(obj, sort_keys=True, indent=4)
    print(text)

with open("config.json", "r") as f:
    config = json.load(f)

STAT_URL = config['url']
headers = config['headers']

#Set query string based on received input: Receive city name
#Find corresponding id based on city
#print("line");
querystring = {"id":"2"}

response = requests.request("GET", STAT_URL + ('teams'), headers=headers, params=querystring)

# Transform response to text form and send to js server
print(response.text)
sys.stdout.flush()